sdktests/server_side_hooks.go

```diff
@@ -269,8 +272,58 @@ func beforeEvaluationDataPropagatesToAfterMigration(t *ldtest.T) {
 	})
 }
 
+// This test is meant to check Requirement HOOKS:1.3.7:
+// The client MUST handle exceptions which are thrown (or errors returned, if idiomatic for the language)
+// during the execution of a stage or handler allowing operations to complete unaffected.
+func errorInBeforeStageDoesNotAffectAfterStage(t *ldtest.T) {
+	const numHooks = 100 // why not?
```

Comment:
Because the test output is not decipherable?

Reply:
Ya know, he's got a point.

Reply:
Thank you for providing a reason :D